src/components/BridgedLocalWrapper.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import React from 'react';
+import { useGUIContext } from '../contexts/GUIContextProvider';
+
+interface Props {
+  children: React.ReactElement | null;
+  blurred?: boolean;
+  showLocal?: boolean;
```

Comment:
Yes. It will be used for showing components only on local .

Reply:
As i mentioned in my comments, it is not implemented yet but it will be used.

Reply:
Well, it's hard to review if it works properly if there is no usage. Also what if it doesn't end up being used and we will simply forget it?

Reply:
i'm using it in a working branch but let's suppose is not used and we forget it, the wrapper will have a feature that can be used some day.
Anyway i will remove it from now to avoid discussions.